vmdb/app/controllers/catalog_controller.rb

```diff
@@ -1389,6 +1390,13 @@ def get_available_resources(kls)
     end
   end
 
+  def default_entry_point
+    @edit[:new][:fqname] =
+      ServiceTemplateOrchestration.default_provisioning_entry_point
```

Comment:
I think you can simply use `class_service_template.default_provisioning_entry_point`